pkg/cmd/tests/options.go

```diff
@@ -164,6 +181,28 @@ func (o *TestFrameworkOptions) Validate(args []string) error {
 		errors = append(errors, fmt.Errorf("gcs-service-account-key-path and s3-credentials-file-path can't be set simultanously"))
 	}
 
+	versionPattern := regexp.MustCompile(`^\d+\.\d+\.\d+(-[a-zA-Z0-9]+)?$`)
```

Comment:
This doesn't cover tag + digest entirely (e.g. latest wouldn't be caught by that), see the following for reference:
- https://github.com/distribution/reference/blob/8c942b0459dfdcc5b6685581dd0a5a470f615bff/reference.go#L6C44-L6C70
- https://github.com/distribution/reference/blob/8c942b0459dfdcc5b6685581dd0a5a470f615bff/regexp.go#L39
- https://github.com/distribution/reference/blob/8c942b0459dfdcc5b6685581dd0a5a470f615bff/regexp.go#L9
I'd import `distribution/reference` to handle that and look there to see how to match and capture both correctly. It might differ between ScyllaDB and manager versions. From the top of my head I'm not sure we handle [tag + ":"] + digest for scylla versions - you'll have to try.